pytorch_lightning/trainer/trainer.py

```diff
@@ -1990,6 +1990,13 @@ def _results(self) -> Optional[ResultCollection]:
     Other
     """
 
+    @property
+    def train_loop(self) -> FitLoop:
+        rank_zero_deprecation(
+            "`Trainer.train_loop` has been renamed to `Trainer.fit_loop` and will be removed in v1.6."
+        )
+        return self.fit_loop
+
```

Comment:
move this up next to the other loop properties in L1956

Reply:
done